locales/ko/템플릿/비즈니스 사례에 대한 의사결정 기록 템플릿/index.md

```diff
@@ -134,4 +134,3 @@
    * 요약
 
    * 세부 사항
-  
```

Comment:
this is another [removing trailing white spaces](https://github.com/joelparkerhenderson/architecture-decision-record/pull/67).
my local workspace has configured like this and it applied when i save the file.
- if i can remember, in future, i would check the diff just after saving a file gonna be edited.
- but i think this minor changes can be done on this time together :)
- or i can do that for the whole files at once if you want.